api/auth.hpp

```diff
@@ -71,37 +80,30 @@ error::Error FetchJWTToken(
 
 class Authenticator {
 public:
-	Authenticator(
-		events::EventLoop &loop,
-		const mender::http::ClientConfig &client_config,
-		const string &server_url,
-		const string &private_key_path,
-		const string &device_identity_script_path,
-		const string &tenant_token = "") :
+	Authenticator(events::EventLoop &loop, chrono::seconds auth_timeout = chrono::minutes {1}) :
```

Comment:
Hmm, should we not hard-code a default here? :thinking:

Reply:
For the timeout, you mean? AFAICT there's no configuration of this value. I wouldn't even make it configurable here in the constructor wasn't it for the tests to avoid long waits. So as much as I don't like hard-coded numbers, this is quite a specific thing.